polynote-frontend/polynote/ui/util/storage.ts

```diff
@@ -73,9 +73,10 @@ type PreferenceListener = (oldValue: Preference, newValue: Preference) => void
 export class Preferences {
     private readonly preferencesKey = "preferences";
     private preferences: Record<string, Preference> = {};
+    private preferenceOptions: Record<string, Record<string, any>> = {};
```

Comment:
I decided to store this separately so that it doesn't get serialized into the JSON, because that annoyed me.